.github/labeler.yml

```diff
@@ -54,7 +54,7 @@ topic-indexing:
   - xarray/core/indexes.py
   - xarray/core/indexing.py
 
-topic-performance:
```

Comment:
Is there a way to do both?

Reply:
You mean use one label to run benchmarks and label it as topic-performance?

Reply:
yes, we'd want both labels applied ideally

Reply:
To apply both labels I would just list them twice, I don't know if there is a neater way.